packages/ui/src/forum/components/PostList/PostList.tsx

```diff
@@ -57,8 +57,9 @@ export const PostList = ({ threadId, isThreadActive, isLoading, replyToPost, isD
   return (
     <Wrapper gap={24}>
       {!isDiscussion && <Pagination pageCount={pageCount} handlePageChange={setPage} page={page} />}
-      {posts.map((post) => (
+      {posts.map((post, index) => (
         <PostListItem
+          index={index}
```

Comment:
The problem is that since the posts are paginated the index 0 might not be the "original thread" but just the first post on page 2. IMO a better approach would be to pass an optional flag like `isPostOrigin` to the `<PostListItem />`

This could be queried through `ForumPost.origin (...on PostOriginThreadInitial)`:
![Screenshot from 2022-05-09 09-56-18](https://user-images.githubusercontent.com/6571453/167366286-c3b6e80c-8b19-442b-a2f3-802255b29c98.png)

(Although there might be a simpler way)

Reply:
@thesan  Maybe it is easier to also use the page number together with the post index to make sure this is the very first post?

Reply:
Page 1:
<img width="1650" alt="image" src="https://user-images.githubusercontent.com/9252017/167489701-879a555b-12d9-422a-8fc5-2f36fe93cf6f.png">
Page 2:
<img width="1669" alt="image" src="https://user-images.githubusercontent.com/9252017/167489975-1e0f8d32-f54d-4a6e-807d-426297df64cf.png">